components/application-operator/cmd/manager/manager.go

```diff
@@ -74,16 +87,31 @@ func main() {
 		log.Fatal(err)
 	}
 
+	log.Printf("Preparing Gateway Manager.")
+
+	gatewayManager, err := newGatewayManager(options, cfg, helmClient)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Printf("Upgrading gateways")
+
+	err = gatewayManager.UpgradeGateways()
```

Comment:
We now call `releaseManager.UpgradeReleases()` and  `gatewayManager.UpgradeGateways()`. I think that `UpgradeReleases` method should be renamed to `UpgradeApplicationReleases` becase Gateways will also be releases.